python/lsst/ap/association/diaPipe.py

```diff
@@ -447,9 +457,12 @@ def run(self,
         diaForcedSources = self.diaForcedSource.run(
             diaCalResult.diaObjectCat,
             diaCalResult.updatedDiaObjects.loc[:, "diaObjectId"].to_numpy(),
+            # Passing ccdExposureIdBits here will take care of warning about
+            # its deprecation.
```

Comment:
It took me a bit to sort out what this comment meant. Maybe rephrase to something like "Deprecation warning about ccdExposureIdBits is handled inside diaForcesource.run()."?